src/doc/src/reference/unstable.md

```diff
@@ -1230,66 +1230,134 @@ cargo check -Z unstable-options -Z check-cfg-well-known-values
 
 ### workspace-inheritance
 
-* RFC: [#2906](https://github.com/rust-lang/rfcs/blob/master/text/2906-cargo-workspace-deduplicate.md)
-* Tracking Issue: [#8415](https://github.com/rust-lang/cargo/issues/8415)
 
-The `workspace-inheritance` feature allows workspace members to inherit fields
-and dependencies from a workspace.
+### The `workspace.package` table
 
-Example 1:
+This would be in [`workspaces.md`][workspaces], under
+[The `workspace.metadata` table][the-workspacemetadata-table]
 
-```toml
-# in workspace's Cargo.toml
-[workspace.dependencies]
-log = "0.3.1"
-log2 = { version = "2.0.0", package = "log" }
-serde = { git = 'https://github.com/serde-rs/serde' }
-wasm-bindgen-cli = { path = "crates/cli" }
-```
+The `workspace.package` table is where you define keys that can be
+inherited by members of a workspace. These keys can be inherited by
+defining them in the member package with `{key}.workspace = true`.
+For Example:
 
 ```toml
-# in a workspace member's Cargo.toml
-[dependencies]
-log.workspace = true
-log2.workspace = true
-```
+# [PROGECT_DIR]/Cargo.toml
+[workspace]
+memebers = ["bar"]
 
-Example 2:
-```toml
-# in workspace's Cargo.toml
 [workspace.package]
 version = "1.2.3"
 authors = ["Nice Folks"]
 description = "..."
 documentation = "https://example.github.io/example"
-readme = "README.md"
-homepage = "https://example.com"
-repository = "https://github.com/example/example"
-license = "MIT"
-license-file = "./LICENSE"
-keywords = ["cli"]
-categories = ["development-tools"]
-publish = false
-edition = "2018"
 ```
 
 ```toml
-# in a workspace member's Cargo.toml
+# [PROGECT_DIR]/bar/Cargo.toml
 [package]
+name = "bar"
 version.workspace = true
 authors.workspace = true
 description.workspace = true
 documentation.workspace = true
-readme.workspace = true
-homepage.workspace = true
-repository.workspace = true
-license.workspace = true
-license-file.workspace = true
-keywords.workspace = true
-categories.workspace = true
-publish.workspace = true
 ```
 
+Keys that are supported:
+`authors`, `categories`, `description`, `documentation`, `edition`,
+`exclude`, `homepage`, `include`, `keywords`, `license`, `license-file`,
+`publish`, `readme`, `repository`, `rust-version`, and`version`.
```

Comment:
Seems like this could be hard to parse as a comma separated list but unsure if any of the alternatives are better
- bulleted list
- table (where column/row is not significant)

Reply:
I think a table may be better with just 2 columns and make it alphabetical like it is now